.github/workflows/post-commit.yml

```diff
@@ -15,12 +15,13 @@
 name: build-branch
 on:
   pull_request:
-    types: [opened, ready_for_review, synchronize]
+    types: [opened, ready_for_review, synchronize, edited]
```

Comment:
This triggers new CI run when PR title or description is edited.  The workflow is then skipped due to the `event.changes.base` condition below.  Previous run is cancelled to avoid duplicate work.  Thus PR will be left without CI workflow.

Reply:
Thanks @adoroszlai for the review. Yes looks like the 'cancel-in-progress'  causes the duplicate one to  get skipped . If we remove the cancellation the previous run won't be aborted  on title update but I guess the that feature was added on purpose.